pages/platform/kubernetes-k8s/label-taint-node-pools/guide.en-asia.md

```diff
@@ -0,0 +1,312 @@
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+  name: hello-world
+  labels:
+    app: hello-world
+spec:
+  type: LoadBalancer
+  ports:
+  - port: 80
+    targetPort: 80
+    protocol: TCP
+    name: http
+  selector:
+    app: hello-world
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: hello-world-deployment
+  labels:
+    app: hello-world
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: hello-world
+  template:
+    metadata:
+      labels:
+        app: hello-world
+    spec:
+      containers:
+      - name: hello-world
+        image: ovhplatform/hello
+        ports:
+        - containerPort: 80
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: nodepool
+                operator: In
+                values:
+                - hourly
+```
+
+```bash
+$ kubectl apply -f deployment.yaml
+service/hello-world created
+deployment.apps/hello-world-deployment created
+```
+
+When you apply this YAML manifest, a Pod will be first in `Pending` state. A new Node will be created. When the new Node will be successfully created, the Pod will be started in this Node.
+
+```bash
+$ kubectl get pod -w
+NAME                                      READY   STATUS    RESTARTS   AGE
+hello-world-deployment-585c6cfcd8-kntp9   0/1     Pending   0          12s
+hello-world-deployment-585c6cfcd8-kntp9   0/1     Pending   0          3m3s
+hello-world-deployment-585c6cfcd8-kntp9   0/1     ContainerCreating   0          3m29s
+hello-world-deployment-585c6cfcd8-kntp9   0/1     ContainerCreating   0          3m41s
+hello-world-deployment-585c6cfcd8-kntp9   1/1     Running             0          3m45s
+```
+
+You have to wait several minutes in order to have a new Node that match with your criterias.
+
+```bash
+$ kubectl get node
+NAME                                         STATUS   ROLES    AGE   VERSION
+hourly-node-d4f9d7                           Ready    <none>   18s   v1.22.2
+nodepool-cc40f90c-effb-4945-b7-node-23e81b   Ready    <none>   21h   v1.22.2
+nodepool-cc40f90c-effb-4945-b7-node-4bd23f   Ready    <none>   21h   v1.22.2
+nodepool-cc40f90c-effb-4945-b7-node-ef121e   Ready    <none>   21h   v1.22.2
+```
+
+And you can check in which node our application is running:
+
+```bash
+$ kubectl get pod -o wide
+NAME                                      READY   STATUS    RESTARTS   AGE   IP         NODE                 NOMINATED NODE   READINESS GATES
+hello-world-deployment-585c6cfcd8-kntp9   1/1     Running   0          13m   10.2.8.2   hourly-node-d4f9d7   <none>           <none>
+```
+
+With this feature you can choose and control where you want to deploy your applications.
+
+## Where do we go from here?
+
+In this tutorial you saw how to create, in your OVHcloud Managed Kubernetes cluster, several kind of node pools and how to deploy your applications easily where we want to.
```

Comment:
"In this tutorial you saw how to create, in your OVHcloud Managed Kubernetes cluster, several kind**s** of node pools and how to deploy your applications easily where **you** want to."